src/vehicle_move.cpp

```diff
@@ -1043,6 +1043,12 @@ veh_collision vehicle::part_collision( int part, const tripoint &p,
                                   critter->get_armor_bash( bodypart_id( "torso" ) );
                 dam = std::max( 0, dam - armor );
                 critter->apply_damage( driver, bodypart_id( "torso" ), dam );
+                if( part_flag( ret.part, "SHARP" ) ) {
+                    critter->make_bleed( effect_source( driver ), bodypart_id( "torso" ), 1_minutes * rng( 1, dam ) );
+                } else if( rng( 1, 20 ) > 15 && dam > 20 ) {
```

Comment:
```suggestion
                } else if( dam > 20 && rng( 1, 20 ) > 15 ) {
```
Doing the simple comparison first can avoid the more expensive random number generation if the simple comparison does not pass.

Reply:
That's a nice tip!